server/src/integration/java/org/opentosca/toscana/core/parse/converter/ModelInstanceConverterIT.java

```diff
@@ -13,19 +13,25 @@
  Tests the conversion of the minimal-docker csar to an effective model
  */
 public class ModelInstanceConverterIT {
-    
+
     private final CsarParseService parser = new CsarParseServiceImpl();
 
     @Test
     public void dockerConverter() throws Exception {
         EffectiveModel model = parser.parse(TestCsars.VALID_MINIMAL_DOCKER_TEMPLATE);
         assertNotNull(model);
     }
-    
+
     @Test
     public void lampNoInputConverter() throws Exception {
         EffectiveModel model = parser.parse(TestCsars.VALID_LAMP_NO_INPUT_TEMPLATE);
         assertNotNull(model);
     }
-    
+
+    // TODO WIP
+//    @Test
+//    public void lampInputConverter() throws Exception {
+//        EffectiveModel model = parser.parse(TestCsars.VALID_LAMP_INPUT_TEMPLATE);
+//        assertNotNull(model);
+//    }
```

Comment:
Won't work yet (#380)

Reply:
is this still a thing even with the "Big Refactor"?

Reply:
Construction of the EffectiveModel after the big refactor looks like this:
```java
EffectiModel model = new EffectiveModel(TestCsars.VALID_LAMP_INPUT_TEMPLATE);
````
After the big refactor branch is applied, this will be uncommented again.